### PR TITLE
Michelle/marketplace webhook

### PR DESCRIPTION
**Pull request by jonmchu**
- created onfleet webhook controller for marketplace objects/listings
- tested marketplace_listing creation through webhook

### CODE REVIEW

File: app/models/marketplace_listing.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class MarketplaceListing < ActiveRecord::Base
```
Comment (amillman) [inline app/models/marketplace_listing.rb]: use `annotate` to document the schema in this file

File: app/controllers/marketplace_webhooks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
+require "active_support/all"
+
+class MarketplaceWebhooksController < ApplicationController
+  skip_before_action :verify_authenticity_token, only: [:index, :create]
+  def index
```
Comment (nhiquach) [inline app/controllers/marketplace_webhooks_controller.rb]: nit: add spacing between validations and methods

File: config/routes.rb [hunk 3]
```diff
@@ -63,6 +64,3 @@
 
   get '*path' => redirect('/')
 end
-
```
Comment (nhiquach) [inline config/routes.rb]: nit: leave one new line at the end

**Comment by nhiquach** [comment]
Looks like in the past, autogenerated .js and .scss files weren't added in. Let's remove those and keep the repo clean since we won't be using them.

**Comment by nhiquach** [comment]
@michellejkwon can you make a migration to delete triggerID/triggername and add a validation to require shortID's presence

**Pull request merged**